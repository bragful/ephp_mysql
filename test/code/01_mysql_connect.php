<?php

$db = mysqli_connect("127.0.0.1", "root", "root", "ephp_mysql");

if (!$db) {
    echo "Error: Cannot connect to MySQL." . PHP_EOL;
    echo "debug errno: " . mysqli_connect_errno() . PHP_EOL;
    echo "debug error: " . mysqli_connect_error() . PHP_EOL;
    exit;
}

echo "Success: Properly connected to MySQL!" . PHP_EOL;
echo "Host info: " . mysqli_get_host_info($db) . PHP_EOL;

mysqli_close($db);
