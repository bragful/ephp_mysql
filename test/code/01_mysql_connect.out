Success: Properly connected to MySQL!
Host info: 127.0.0.1 via TCP/IP
